packages/orchestrator/internal/sandbox/network/firewall.go

```diff
@@ -146,9 +151,17 @@ func (fw *Firewall) installRules() error {
 }
 
 // AddBlockedIP adds a single CIDR to the block set at runtime.
-func (fw *Firewall) AddBlockedIP(cidr string) error {
+func (fw *Firewall) AddBlockedIP(address string) error {
```

Comment:
We switched to inserting blocked IPs and not using CIDRs, which seems cleaner to me.